app/controllers/api/v1/users/gym_members_controller.rb

```diff
@@ -5,7 +5,7 @@ def index
     return error_message if params[:yelp_gym_id].blank?
 
     user = User.find(params[:user_id])
-    users = user.non_friends_at_same_gym(params[:yelp_gym_id])
+    users = user.non_followees_at_same_gym(params[:yelp_gym_id])
```

Comment:
Thanks for updating the naming of these!!

Reply:
For sure!  It was getting so confusing as I was building it haha, the cleanup definitely helped demystify the relationships a bit!